src/base-components/checkbox-base/index.js

```diff
@@ -60,9 +61,11 @@ const CheckboxBase = React.forwardRef(({
           {...inputProps}
         />
       </span>
-      <span className='self-center align-middle'>
-        <label className={classes.label} htmlFor={inputID}>{label}</label>
-      </span>
+      {showLabel && 
```

Comment:
redundant prop, can check `label`:
`{label && ...}`

Reply:
for checkbox label is required without this prop I won't be able to use the component without label